_posts/2023-10-26-louisvillejefferson-county-metro-government-balance-kentucky/louisvillejefferson-county-metro-government-balance-kentucky.Rmd

```diff
@@ -0,0 +1,135 @@
+| Black or African American | 23.8%      |
+| Hispanic or Latino        | 6.5%       |
+| Asian                     | 2.7%       |
+
+![Figure 1: This map shows the proportion of minorities within each precinct in Louisville. Darker shades of blue indicate a higher proportion of minorities, including Black, Hispanic, and Asian populations, living in the precinct compared to the majority, White population. The thinner black lines represent precinct boundaries, while the thicker white lines represent current district boundaries.](images/minority.png){fig-align="center" width="80%"}
+
+### Louisville Metro Council
+
+The Louisville Metro Council comprises 26 members who are elected from the city's 26 districts in partisan elections. Council members serve staggered four-year terms. Since the merger with Jefferson County in 2003, the council has consistently maintained a Democratic majority. The distribution of seats held by each party is as follows (Democrats: Republicans): 15:11 (2006), 17:9 (2010), 17:9 (2014), 19:7 (2018), and 17:9 (2022)[^1].
+
+[^1]: All information regarding the elections in Louisville is based on the official statements of votes cast available at the website below. Jefferson County Clerk Election Center, *Election Results*, <https://elections.jeffersoncountyclerk.org/election-results/>
+
+### Mayoral Elections
+
+Democratic candidates have also consistently won in mayoral elections in Louisville. The margins between Democratic and Republican candidates have usually been substantial, often ranging from 30 to 40 percent. However, these elections can still appear competitive. For example, in the 2022 mayoral election, the margin between both candidates was only 6 percent. Additionally, the margin in the 2010 mayoral election was narrower, with a difference of just 3 percent.
+
+| Year | DEM                     | REP                    |
+|------|:------------------------|:-----------------------|
+| 2006 | Jerry Abramson (67.2%)  | Kelly Downard (31.3%)  |
+| 2010 | Greg Fischer (51.1%)    | Hal Heiner (48.4%)     |
+| 2014 | Greg Fischer (68.5%)    | Bob DeVore (31.3%)     |
+| 2018 | Greg Fischer (61.0%)    | Angela Leet (36.3%)    |
+| 2022 | Craig Greenberg (51.7%) | William Dieruf (46.2%) |
+
+![Figure 2: This map displays precinct-level voting shares in Louisville (2022 Mayoral Election). A darker shade indicates a larger number of voters supporting either of the parties within the precinct. The black lines represent the current district boundaries, each marked with its corresponding district number.](images/precincts_voting_behavior.png){fig-align="center" width="80%"}
+
+## Competitiveness in Metro Council Elections
+
+Metro Council elections in Louisville appear to be relatively competitive, even with the strong Democratic presence. For example, in 2022, two Democrats lost their seats, resulting in a shift in the Metro Council's composition from 19 Democrats and 7 Republicans to 17 Democrats and 9 Republicans.
```

Comment:
Might be nice to start off with a definition of competitiveness for readers to understand. At a high level )not mathematical), what indicates a district that is more vs. less competitive?